### PR TITLE
Eln branch

### PR DESCRIPTION
**Pull request by tdawson**
How to request an ELN Branch

--signed off by Troy Dawson <tdawson@redhat.com>

### CODE REVIEW

File: modules/ROOT/pages/eln-branch.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+....
+
+== Step 3: Maintain Branch
+
+After your branch has been created, remember that the ELN branch is your
+responsibility.  Do you create the branch, build it once, and then leave
+the branch to rot.
+
+*If you decide that you no longer need the ELN branch, you must tell the
+ELN SIG before retiring the branch.* The ELN SIG will need to change
```
Comment (yselkowitz) [inline modules/ROOT/pages/eln-branch.adoc:67]: Normally, retiring a branch also blocks packages from being built in the corresponding koji tag/target.  Obviously that must not happen in the case of ELN, as *only* the dist-git branch is no longer being used, but the package will (unless being dropped from RHEL/ELN entirely) continue being built by DBS (once the config is updated).  If the procedure for "retiring" an ELN branch differs as a result, that needs to be documented.

File: modules/ROOT/pages/eln-branch.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+If you do not think you will be able to do this in a timely manner
+and/or would like me to be a co-maintainer, please add me (FAS <your FAS Id>)
+as a co-maintaine through https://src.fedoraproject.org/rpms/<package>/adduser
+....
+
+== Step 3: Maintain Branch
+
+After your branch has been created, remember that the ELN branch is your
+responsibility.  Do you create the branch, build it once, and then leave
+the branch to rot.
```
[the remark this answers is not in the document]
Comment (tdawson) [reply modules/ROOT/pages/eln-branch.adoc]: Sorry about that.  My fingers didn't type what my mind thought.

File: modules/ROOT/pages/eln-branch.adoc (hunk 1 above)
Comment (tdawson) [reply modules/ROOT/pages/eln-branch.adoc]: You are correct.  We haven't talked about how to retire an ELN branch.  For now lets say that the sig will retire them.

Comment (yselkowitz) [reply modules/ROOT/pages/eln-branch.adoc]: I'd go a step further and explicitly say NOT to `fedpkg retire` the branch.

**Pull request merged**